docs/Developers/Rankings.rst

```diff
@@ -0,0 +1,103 @@
+Rankings
+========
+
+The coala community is split into different ranks which have different rights
+and obligations. This page will explain which the ranks are and what you need
+to do to get promoted to a higher rank.
+
+Currently the ranks can be seen `here <https://github.com/orgs/coala/teams>`__
+and are as it follows:
+
+ - newcomers
+ - developers
+ - maintainers
+ - admins
+
+The Newcomer Rank
+-----------------
+
+Everyone starts here. This is the rank where you can get a promotion easiest
+and which has the lowest obligations, and, obviously, the lowest rights.
+
+Rights
+~~~~~~
+
+The newcomers have no write access, and no direct labelling rights. The only
+thing that can be done by them is reviewing by leaving useful comments on
+Pull Requests which help them prove themselves as good reviewers.
+
+How to get promoted
+~~~~~~~~~~~~~~~~~~~
+
+If you complete the following tasks, you will become a developer at coala:
+
+  - run coala on a project of yours
+  - merge a ``difficulty/newcomer`` Pull Request
+  - review at least a ``difficulty/newcomer`` Pull Request
+  - merge a ``difficulty/low`` Pull Request
+  - review at least a ``difficulty/low`` or higher Pull Request
+
+When you ran coala on a project, please fill our
+`usability survey <http://coala.io/usability>`_. Once you got your first Pull
+Request merged successfully, fill in our
+`survey form <http://coala.io/newform>`_. With that you can help us making your
```

Comment:
well this was copy pasted from what someone else (think lasse) written on newcomer's guide, but ill modify it :D